src/JobDefinitionParser.php

```diff
@@ -45,7 +45,13 @@ public function createJobDefinitionsForJob(
                 $configuration = $components->getConfiguration($job->getComponentId(), $job->getConfigId());
                 /** @var array $configuration */
 
-                $this->checkUnsafeConfiguration($component, $configuration, $job->getBranchType() ?? BranchType::DEV);
+                if (!$clientWrapper->getClientOptionsReadOnly()->useBranchStorage()) {
```

Comment:
Je to vypnutý, protože v sox projektu je to řešený přes vault a na UI už je to schovaný